src/index.ts

```diff
@@ -264,6 +301,73 @@ export default class Client extends BaseClient {
     return Workflow.fromResponse(result);
   }
 
+  async triggerWorkflow(
+    workflowId: string,
+    triggerType: TriggerType,
```

Comment:
The manual trigger data needs to checked based on the TriggerType of the workflow, so this workflow trigger type is mandatory for the caller, otherwise we need to make another call to get the workflow data.